rfc2html.py

```diff
@@ -402,7 +402,7 @@ def appendix_replacement(match):
 
     #
     #text = re.sub("\f", "<div class=\"newpage\" />", text)
-    text = re.sub("\n?\f\n?", "</pre>\n<hr class='noprint'/><!--NewPage--><pre class='newpage'>", text)
+    text = re.sub("\n?\f\n?", '\n', text)
```

Comment:
Right now, a using app (like the datatracker) would have to provide css, and maybe there should be some discussion in the docs here about expectations for the new semantic class names. With the horizontal rule  moved to "divider", will the datatracker already style it as display:none in print media? Or does should this be considered a breaking change here that needs coordination there?

Reply:
The movement of the tag should render fine, and doesn't require DT changes _if_ we keep the classes unchanged.

Reply:
I don't think that answers my question (maybe I'm not getting it) - what will keep this from introducing a horizontal rule when printing that wasn't there before?

Reply:
If we keep `noprint`, which CSS exists for already in the consumers of the library, that will make the right thing happen.

Reply:
I switched it back to the current CSS class names.